core/new-gui/src/app/workspace/component/visualization-panel-content/visualization-panel-content.component.spec.ts

```diff
@@ -104,9 +104,25 @@ describe('VisualizationPanelContentComponent', () => {
     spyOn(workflowStatusService, 'getCurrentResult').and.returnValue(testData);
 
     component.operatorID = 'operator1';
-    component.ngAfterViewInit();
+    component.ngAfterContentInit();
 
     expect(component.generateSimpleScatterplot).toHaveBeenCalled();
   });
 
+  it('should draw a sample html', () => {
+    const testData: Record<string, ResultObject> = {
+      'operator1': {
+        operatorID: 'operator1', chartType: ChartType.HTML_VIZ,
+        table: [{'HTML_content': '<div>sample</div>'}], totalRowCount: 1
```

Comment:
I think you will need to change here, if you change the column name to `HTML-content` on the backend.